pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1210,11 +1210,15 @@ func GetSeccompOpts(annotations map[string]string, ctrName, profileRoot string)
 		return nil, nil
 	}
 
-	if !strings.HasPrefix(profile, "localhost/") {
+	if !strings.HasPrefix(profile, "localhost/") && !strings.HasPrefix(profile, "/") {
```

Comment:
Why changing the dockertools implementation? This change should not affect the existing docker implementation.